src/SmartComponents/Systems/SystemTable.test.js

```diff
@@ -27,9 +27,11 @@ const initStore = (state) => {
     return mockStore(state);
 };
 
+const apply = () => {};
```

Comment:
Explain.

Reply:
Since `apply` is now used inside the `SystemsTable` (before, it was just passed along to other functions), the tests fail if `apply` is undefined. thus i'm just passing a plain function into the component

Reply:
We will need to make this work even when no `apply` is provided.